### DISCUSSION TITLE
How can i search HtmlBody or Subtitle or etc in Admin?

### DISCUSSION THREAD

**Discussion by smemis** [OrchardCMS/OrchardCore#13522] [General]
<img width="1915" alt="image" src="https://user-images.githubusercontent.com/4637285/229653946-8c67c5db-cc64-4e09-a989-1facee6f8be1.png">

**Comment by hishamco** [comment]
Did you enable FullTextSearch in both fields / parts

**Reply by smemis** [reply]
Yes absolutelly

**Comment by ns8482e** [comment]
I think default term for admin search is title. I am not hundred percent sure but by providing term `body:` should work.

However you can always add your custom yessql filter https://github.com/sebastienros/yessql/wiki/Query-Filters#yessqlfiltersquery

**Comment by smemis** [comment]
I can able search on the frontend of the site, but not on the backend admin Content Items.
/Admin/Contents/ContentItems?admin=591330047

**Reply by ns8482e** [reply]
Correct admin search is based on YesSql filter - see above answer for yessql filter doc link

**Reply by smemis** [reply]
hmm a new challenge begins for me

**Reply by smemis** [reply]
How to write a filter for fields containing text in general, rather than writing a filter for each content type separately

**Comment by smemis** [comment]
I've been trying all day but I couldn't succeed. Please can someone help?

I can search with the fields I have included in the fulltext search on the frontend, but it does not work on the admin side.

Shouldn't Admin Content Items also be able to search by fields included in the fulltext?
What code should I write to achieve this?
